Docs: TMC location config (#141)
terramate-io/terramate-docs: cli/reference/configuration/index.md

More details can be found [here](../../projects/configuration.md#the-terramateconfigrunenv-block).

> ## terramate.config.cloud.location 

Reviewer: ```suggestion
## terramate.config.cloud
```

> The `terramate.config.cloud.location` block has no labels and is used to set the Terramate Cloud region with the following schema:

Reviewer: ```suggestion
The `terramate.config.cloud` block has no labels and is used to set the Terramate Cloud configuration with the following schema:
```

| name             |      type      | description | default |
|------------------|----------------|-------------|---------|

> | us | string | "us" or "eu" | "eu"

Reviewer: ```suggestion
| location | string | "us" or "eu" | "eu"
```
